### ISSUE TITLE
Why is ChatEnterCallback triggered when receiving group invites?

### ISSUE THREAD

**Issue by kraxarn** [SteamRE/SteamKit#391]
Why is `SteamFriends.ChatEnterCallback` triggered when getting invited to a group or accepting a group invite from Steam Community? Is there any way to check if the user is actually entering a chatroom or getting invited to a group?

**Comment by VoiDeD** [comment]
Does `EnterResponse` or `ChatFlags` expose anything obvious? If not, you may need to look into using NetHook and seeing if the payload of the `MsgClientChatEnter` message gives you any useful info.

**Comment by KraXarN** [comment]
The callback is the same as when it joined the last chat room.

**Comment by yaakov-h** [comment]
Closing due to inactivity. I don't believe this is an issue with SteamKit, it's probably something to do with the underlying (undocumented) Steam protocol.

**Comment by kraxarn** [comment]
So I decided to look into the issue again, not sure if I should open a new issue or not.

I tried hooking NetHook2 into the Steam client and tried to first invite myself to a group and then join a chatroom. I'm not too familiar with NetHook2 and the Analyzer, but in the list I can see a message called `ClientChatEnter`, but only when joining the chat. When being invited to the group I only get a `ClientClanState` message.
